src/NHibernate.Test/NHSpecificTest/NH3438/BaseFixture.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections;
+using System.Collections.Generic;
+using NHibernate.Hql.Ast.ANTLR;
+using NHibernate.Util;
+
+namespace NHibernate.Test.NHSpecificTest.NH3438
```

Comment:
Typo, it should be 3488. All files and the older of the test cases for NH3438 have this typo. (Already renamed on my local branch.)